bridge/src/observers/burn-event-observer.ts

```diff
@@ -49,12 +52,28 @@ export class EthereumBurnEventObserver implements IObserver<{ blockHash: BlockHa
                     channel: "#nine-chronicles-bridge-bot",
                     ...new UnwrappedEvent(this._explorerUrl, this._etherscanUrl, sender, recipient, amountString, nineChroniclesTxId, transactionHash).render()
                 });
+                await this._opensearchClient.to_opensearch("info", {
+                    content: "wNCG -> NCG request success",
+                    libplanetTxId: nineChroniclesTxId,
+                    ethereumTxId: transactionHash,
+                    sender: sender,
+                    recipient: recipient,
+                    amount: amountString,
+                });
```

Comment:
> 잘 동작한다면, opensearch call 을 async 하게 변경하고 싶습니다.

`await` 하고 싶지 않고 `fire and forget` 하고 싶다는 뜻인가요? 그냥 `await` 을 지우는 것으로도 해결이 가능해 보입니다.

Reply:
@riemannulus exactly - removing `await` would just work, right? I would like to make sure of them before I remove `await`. I put `await` here just in order to verify the functionality. Once your review is completed, I will make an additional commit to remove `await`.

Reply:
@riemannulus @moreal plus, shouldn't we remove `await` from Slack calls as well? Just in case Slack API goes wrong or something - the bridge should work no matter Slack or OpenSearch is working or not. If so, I will make another ticket or work on it after the current PR gets merged and deployed to prod.

Reply:
> shouldn't we remove await from Slack calls as well?

AKAIR it was some tasks handled by slack calls. So it was `await` to end the call. But now, I agree with your opinion. It is unnecessary now. Thanks if you make another ticket about this issue 👍

Reply:
I will take care of removing 'await' from Slack calls then - thanks!

Reply:
FYI: Asana ticket for ^ https://app.asana.com/0/1200937754490028/1202296376044998/f